src/popper/commands/cmd_translate.py

```diff
@@ -0,0 +1,48 @@
+import click
+import os
+
+from popper.cli import pass_context, log
+from popper.parser import WorkflowParser
+from popper.translators.translater import WorkflowTranslator
+
+
+@click.command(
+    "translate",
+    short_help="Translate workflow configuration files of different runners.",
+)
+@click.option(
+    "infmt",
+    "--from",
+    help="Format of the input file.",
+    type=click.Choice(["popper"]),
+    default="popper",
+)
+@click.option(
+    "infile",
+    "--file",
+    "-f",
+    help="File containing the definition of the workflow.",
+    required=False,
+    default=".popper.yml",
+)
+@click.option(
+    "outfmt",
```

Comment:
maybe `to_fmt` and `from_fmt`

Reply:
resolved 5c358a324021e98425e9bd11b35219fdb66d558f